### ISSUE TITLE
Problem: refactor owTypes into separate files

### ISSUE THREAD

**Issue by foreseaz** [crypto-com/chain-nodelib#214]
**Is your feature request related to a problem? Please describe.**
It's great to define proper owTypes for each module like `owUnbondTransactionBuilderOptions` for staking. But it's a little confusing when having the first glance searching for the `real` types and interfaces.

**Describe the solution you'd like**
For discussion, is it better to separate all owTypes into, for example, `ow_types.ts`, in each module, or globally.

**Comment by calvinlauco** [comment]
Yes, it was a hard decision when I first decide to put `ow` types with the TypeScript types together. The major reason is I want to keep the two type definitions together such that they will be consistent with each other when being changed.

But I understand when things grow big this becomes a confusion. If I must choose I will prefer to keep a per module `owTypes.ts` file instead of a global one so that two types are kept close to each other.

**Comment by foreseaz** [comment]
Agree, delicate `owTypes.ts` for each module should be good.
